sysutils/mas/Portfile

```diff
@@ -16,9 +16,11 @@ maintainers         {kimuraw @kimuraw} openmaintainer
 platforms           darwin
 
 depends_build       port:carthage
-checksums           md5     ca1560e390fa17988b80248ec7b781b3 \
-                    sha1    e76df7b060cad78554e17ae328e55040d1b2698f \
-                    sha256  50af96b34ade6f0b67a8564e19d2b8818d56186329ce9a5d3f1b0ad3e9df8f6d
+checksums           md5     8222d2f918358434a7cbb3f9c4690d77 \
+                    sha1    ea4381cccc9d8938d444bbb2f456ebbed6f3d47d \
```

Comment:
md5/sha1 are obsolete, should be removed

Reply:
Okay, I remove MD5/SHA1 checksums.